HOW_TO_RELEASE.md

```diff
@@ -18,7 +20,7 @@ An automatic GitHub action will start to push and publish the new release to PyP
 - Before releasing, you need to delete **both** the tag and the release of the previous release. If you release with the same tag without deletion, it will ignore your commit changing the version number, and PyPI will block the upload again. You're stuck in a circle.
 
 
-## The hard way
+### The hard way
 
  1. Go to your local main repository (not the fork) and ensure your master branch is synced:
        git checkout master
```

Comment:
actually, master does not exist anymore, it should be "main".

Reply:
Also done!